config/internal/common/default/mariadb-access-policy.yaml.tmpl

```diff
@@ -0,0 +1,23 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: mariadb-{{.Name}}
+  namespace: {{.Namespace}} 
+spec:
+  podSelector:
+    matchLabels:
+      app: mariadb-{{.Name}}
+      component: data-science-pipelines      
+  ingress:
+    - ports:
+        - protocol: TCP
+          port: 3306    
+      from:
+        - podSelector:
+            matchLabels:              
+              app.kubernetes.io/name: data-science-pipelines-operator
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: redhat-ods-applications
```

Comment:
also add `opendatahub` namespace

Reply:
actually a better way to do this would be to pass the DSPO namespace as an environment variable, then use then use that parameterize value here

you can add the dspo namespace using downward api, ([example](https://jamesdefabia.github.io/docs/user-guide/downward-api/))  add it [here](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/manager/manager.yaml#L38)

Reply:
fwiw i wonder if it's worth it to enrich DSPO with a template/variable for its own namespace, such that we could just use something like `{{.DSPO.Namespace}}`

Reply:
Sure,  missed to see Humair's previous comment too !